compress/lz4.c

```diff
@@ -50,10 +51,34 @@ static void *compr_lz4_open(void)
   struct ComprLz4Ctx *ctx = mutt_mem_malloc(sizeof(struct ComprLz4Ctx));
 
   ctx->buf = mutt_mem_malloc(LZ4_compressBound(1024 * 32));
+  ctx->level = 1;
 
   return ctx;
 }
 
+/**
+ * compr_lz4_set_level - Implements ComprOps::set_level()
+ */
+static void compr_lz4_set_level(void *cctx, short level)
+{
+  if (!cctx)
+    return;
+
+  struct ComprLz4Ctx *ctx = cctx;
+
+  if (level < 1 || level > 12)
+    mutt_error("Wrong Compression Level for LZ4!");
```

Comment:
A few points:
- If the compression level is out of range, should we clamp the value?

- Add constants for the minimum, default and maximum compression levels

- Please can we change the message to help the user, e.g.
  ```c
  mutt_warning(_("The compression level for %s should be between %d and %d"),
               ctx->name, MIN_COMP_LEVEL, MAX_COMP_LEVEL);
  ```

Message:
- Tell the user the limits
- Reduced to a warning, because we're telling the user what to do next time
- Removed the `!` so it's less SHOUTY
- Add the gettext translation macro `_("text")`
- Made the message generic so it can be shared by all three compression methods
  (reducing the work for translators)

Reply:
Will do so.